.github/workflows/update-changelog.yml

```diff
@@ -35,5 +35,9 @@ jobs:
       - name: Push changes
         run: |
           git add CHANGELOG.md
-          git commit -m "Post-release updates [auto]"
-          git push
+          git commit --amend --no-edit
+          GIT_TAG_MESSAGE=$(git tag -l --format='%(contents)' ${{ github.ref_name }})
+          git tag -d ${{ github.ref_name }}
+          git tag -a ${{ github.ref_name }} -m "$GIT_TAG_MESSAGE"
+          git checkout master
+          git push --tags --force
```

Comment:
Why is a force push needed?

Reply:
The tag that we intend to rewrite will exist in the remote. The lack of a force push will give the error `already exists`. So to overwrite the tag we need a force push